### PR TITLE
Move commits inside the `action` div

### PR DESCRIPTION
**Pull request by xphoniex**
Fixes 4th item of issue #745 

Screenshot from 2023-05-28 14-59-55

### CODE REVIEW

File: src/components/Comment.svelte [hunk 1]
```diff
@@ -37,7 +37,7 @@
   }
   .card-body {
     font-size: var(--font-size-small);
-    padding: 0 1rem 1rem 1rem;
+    padding: 0 1rem 0 1rem;
```
Comment (xphoniex) [inline src/components/Comment.svelte:40]: Notice ^

Looked around and nothing seems to be broken by it.

If we don't set it to zero we get an ugly gap:

Screenshot from 2023-05-28 15-10-12
Screenshot from 2023-05-28 15-09-56

**Comment by xphoniex** [comment]
With icon for commits (fixes no. 5 of issue https://github.com/radicle-dev/radicle-interface/issues/745)

Screenshot from 2023-05-28 19-58-27

**Comment by xphoniex** [comment]
Screenshot from 2023-06-02 08-56-13

**Comment by xphoniex** [comment]
Screenshot from 2023-06-02 09-20-29

**Review by rudolfs — APPROVED** [review]

**Pull request merged**